dianna/visualization/image.py

```diff
@@ -10,10 +10,18 @@ def determine_vmax(max_data_value):
     return vmax
 
 
-def plot_image(heatmap, original_data=None, heatmap_cmap=None, data_cmap=None):
+def plot_image(heatmap, original_data=None, heatmap_cmap=None, data_cmap=None, show_plot=True, output_filename=None):
     """
     Example image figure
+    :param heatmap:
+    :param original_data:
+    :param heatmap_cmap:
+    :param data_cmap:
+    :param show_plot: Shows plot if true (for testing or writing plots to disk instead)
+    :param output_filename: Name of the file to save the plot to (optional).
+    :return:
```

Comment:
We use Google-style docstrings in the rest of the code, could you update this to Google style as well?